### ISSUE TITLE
Querying related tables for multiple records

### ISSUE THREAD

**Issue by kamalsivalingam** [JasperFx/marten#2643]
Hi,
I saw examples of querying related tables using LINQ in Marten for a single entity.
Is it possible to get multiple records from a table and their related table records using LINQ on IQueryable?

**Comment by jeremydmiller** [comment]
Yep. See this: https://martendb.io/documents/querying/linq/include.html
